src/main/webapp/resources/js/pages/projects/samples/linker/Linker.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { render } from "react-dom";
+import { Button, Modal, Typography } from "antd";
+import { CodeOutlined } from "@ant-design/icons";
+import { grey2, grey4, grey9 } from "../../../../styles/colors";
+import { getNGSLinkerCode } from "../../../../apis/linker/linker";
+import { SPACE_SM } from "../../../../styles/spacing";
+
+const { Paragraph, Text } = Typography;
+
+/**
+ * React component to display the ngs-linker command to a user based
+ * on the currently selected samples.
+ * @returns {*}
+ * @constructor
+ */
+function Linker() {
+  function handleSampleIds(e) {
+    // Post data to get linker command.
+    const { detail } = e;
```

Comment:
Change detail to a more descriptive variable name

Reply:
Nope this is what it is called in the `CustomEvent` from the browser.

Reply:
Sounds good